src/toolchain/llvm.rs

```diff
@@ -53,6 +55,14 @@ impl Llvm {
         llvm_path
     }
 
+    fn get_bin_path(&self) -> String {
```

Comment:
```suggestion
    /// Gets the binary path of clang
    fn get_bin_path(&self) -> String {
```

Reply:
I'm not sure I quite grok the meaning of using bin on Windows in the first place, I'll prefer to just keep it as-is unless you feel strongly